cmd/pricer/main.go

```diff
@@ -17,10 +17,10 @@ import (
 
 	"github.com/mysteriumnetwork/discovery/config"
 	_ "github.com/mysteriumnetwork/discovery/docs"
+	"github.com/mysteriumnetwork/discovery/middleware"
 	"github.com/mysteriumnetwork/discovery/price"
 	"github.com/mysteriumnetwork/discovery/price/pricing"
 	"github.com/mysteriumnetwork/discovery/price/pricingbyservice"
-	"github.com/mysteriumnetwork/discovery/token"
```

Comment:
I see this token is not modified, isn't it unused now and can be deleted?

Reply:
True. Removed